file-formats/ddls/ddls.json

```
@@ -0,0 +1,126 @@
+
+{ 
+  "$schema": "https://json-schema.org/draft/2020-12/schema", 
+  "$id": "http://sap.com/schema/nrob.json", 

```

Id should be the actual location of the schema
```suggestion
  "$id": "https://github.com/SAP/abap-file-formats/blob/main/file-formats/ddls/ddls.json",
```

---

Thanks. Fixed with next commit.